configure.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+#
+# Copyright 2021  CEPrE-Unilurio 
+
+# export microservices packages to PYHTONPATH
+
+cp ~/.bashrc ~/.bashrc.back
+cp ~/.bashrc ~/.bashrc.back.back
+rm ~/.bashrc 
+python scripts/python_path.py
```

Comment:
This is in `resource/` folder, not `scripts/`. I'm guessing the best option would be to move the file to the `scripts/` folder?

I think there's a problem with this. You are asking the user to run a python script without installing the virtual environment before... And that file uses f-strings which were only introduced in python 3.6, therefore there's a bit of a risk here of things breaking. Shouldn't you install a virtualenv first?

Reply:
No, the configure is meant to be in the same level as Makefile, because we build full path from current dir ... so it more easy that to place it at the top level dir 
our projects are using python 3.6+, so that is why I've used f-string it is more straightforward

Reply:
I don't think it's a good idea to have the option to run a script that requires a specific python version (ie >3.6) without having an environment for that. The environment should be exactly for these cases. 

I guess then I'm not understanding, I trust your judgement but we should discuss in our next meeting, then.